joint_state_publisher/joint_state_publisher/joint_state_publisher.py

```diff
@@ -59,6 +59,47 @@ def _init_joint(self, minval, maxval, zeroval):
             joint['effort'] = 0.0
         return joint
 
+    def init_sdf(self, robot):
+        robot = robot.getElementsByTagName('model')[0]
+        # Find all non-fixed joints
+        for child in robot.childNodes:
+            if child.nodeType is child.TEXT_NODE:
+                continue
+            if child.localName == 'joint':
```

Comment:
I know the rest of the code isn't like this, but I think this would be easier to read if we got out early.  That is, change this line to be:

```
if child.localName != 'joint':
    continue
```

and then unindent the rest of the code one level (we should do the same for `init_urdf` and `init_collada`, but that should be a separate PR).

Reply:
https://github.com/ros/joint_state_publisher/pull/94